src/sql/src/func.rs

```diff
@@ -1693,7 +1694,10 @@ lazy_static! {
                 params!(String, Timestamp) => BinaryFunc::TimezoneTimestamp, 2069;
                 params!(String, TimestampTz) => BinaryFunc::TimezoneTimestampTz, 1159;
                 // PG defines this as `text timetz`
-                params!(String, Time) => BinaryFunc::TimezoneTime, 2037;
+                params!(String, Time) => Operation::binary(|ecx, lhs, rhs| {
+                    let wall_time = DateTime::<Utc>::from(ecx.qcx.scx.pcx.wall_time).naive_utc();
+                    Ok(lhs.call_binary(rhs, BinaryFunc::TimezoneTime{wall_time}))
```

Comment:
Is this the canonical way to do this? I'm not super clear on how this works and I just cargo-culted stuff until it compiled.

Reply:
Yeah seems good to me.